microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Updates to Region information exposed to web content
+description: How customers and web developers can use the OS regional format in Microsoft Edge for improved site experiences
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/11/2022
+--- 
+# Updates to Region information exposed to web content
+
+## Abstract 
```

Comment:
```suggestion
```
I will mainly to a local rewrite of article in VS Code then push a commit, even though I might put Comments/Suggestions here in github.com PR UI too.

Reply:
I am adding item on here directly to the .md file.